### PR TITLE
Andrewzhang

### PR DESCRIPTION
**Pull request by andrew123zhang**
Datadownload and s3interface running
Still working on/writing tests for batchbalst, jobhandling/jobworker/jobcreating

### CODE REVIEW

File: src/datamanagement/Datadownload.py [hunk 1]
```diff
@@ -47,11 +47,11 @@ def ftpConnect(siteHome, siteSubDir):
     return ftp
 
 
-def ftpDownload(siteHome, siteSubDir, dbFile, regexEnding, ftp):
+def ftpDownload(siteHome, siteSubDir, dbFile, fileRegex, ftp):
```
Comment (glicxu) [inline src/datamanagement/Datadownload.py:50]: can you explain the reason?

**Review by glicxu — COMMENTED** [review]
Please add more comment to the reasons for the changes

File: src/datamanagement/Datadownload.py (hunk 1 above)
Comment (andrew123zhang) [reply src/datamanagement/Datadownload.py:50]: example resolution asdfsdfsd

**Review by glicxu — APPROVED** [review]
looks good

**Pull request merged**